app/assets/javascripts/misc/i18n-strings.js.erb

```diff
@@ -0,0 +1,14 @@
+window.LoginGov = window.LoginGov || {};
+
+<% keys = [
+  'instructions.password.strength.i',
+  'instructions.password.strength.ii',
+  'instructions.password.strength.iii',
+  'instructions.password.strength.iv',
+  'instructions.password.strength.v',
+] %>
+
+window.LoginGov.i18n = {};
+<% keys.each do |k| %>
+  window.LoginGov.i18n['<%= k %>'] = '<%= I18n.t(k) %>';
```

Comment:
I prefer `key` to `k` if that's alright with you -- I think Reek generally does not like single-letter vars, either.